frontend/src/scenes/insights/filters/ActionFilter/ActionFilter.tsx

```diff
@@ -76,159 +76,154 @@ export interface ActionFilterProps {
     }: Record<string, JSX.Element | string | undefined>) => JSX.Element
 }
 
-export const ActionFilter = React.forwardRef<HTMLDivElement, ActionFilterProps>(
-    (
-        {
-            setFilters,
-            filters,
-            typeKey,
-            addFilterDefaultOptions = {},
-            mathAvailability = MathAvailability.All,
-            buttonCopy = '',
-            disabled = false,
-            sortable = false,
-            showSeriesIndicator = false,
-            seriesIndicatorType = 'alpha',
-            hideFilter = false,
-            hideRename = false,
-            hideDuplicate = false,
-            propertyFiltersPopover,
-            customRowSuffix,
-            entitiesLimit,
-            showNestedArrow = false,
-            actionsTaxonomicGroupTypes,
-            propertiesTaxonomicGroupTypes,
-            hideDeleteBtn,
-            renderRow,
-            buttonType = 'tertiary',
-            readOnly = false,
-            bordered = false,
-        },
-        ref
-    ): JSX.Element => {
-        const { currentTeamId } = useValues(teamLogic)
-        const logic = entityFilterLogic({
-            teamId: currentTeamId,
-            setFilters,
-            filters,
-            typeKey,
-            addFilterDefaultOptions,
-        })
-        const { reportFunnelStepReordered } = useActions(eventUsageLogic)
+export const ActionFilter = React.forwardRef<HTMLDivElement, ActionFilterProps>(function ActionFilter(
```

Comment:
That's for the ESLint rule `react/display-name` - with the arrow function inside `forwardRef`, the wrapped component is unnamed.

Reply:
Oh interesting, I thought we were already compliant with this rule. Looks like it's just been a while since this file was last edited and only now did ESLint catch this.